lib/srv/mcp/stdio_test.go

```diff
@@ -307,4 +308,14 @@ func TestHandleSession_execMCPServer(t *testing.T) {
 			}
 		})
 	}
+
+	t.Run("reporting", func(t *testing.T) {
+		// Note that some metrics may be incremented by other tests too so here
+		// just checking they are non-zero.
+		require.Positive(t, testutil.ToFloat64(setupErrors.WithLabelValues(types.MCPTransportStdio)))
+		require.Positive(t, testutil.ToFloat64(accumulatedSessions.WithLabelValues(types.MCPTransportStdio)))
+		require.Positive(t, testutil.ToFloat64(messagesFromClient.WithLabelValues(types.MCPTransportStdio, "request", "initialize")))
+		require.Positive(t, testutil.ToFloat64(messagesFromClient.WithLabelValues(types.MCPTransportStdio, "notification", "notifications/initialized")))
+		require.Positive(t, testutil.ToFloat64(messagesFromServer.WithLabelValues(types.MCPTransportStdio, "response", "initialize")))
+	})
```

Comment:
This depends on other tests being run, but doesn't check that was the case. Other tests may be skipped due to configuration or still running (in the future, if anyone enables parallel runs).